packages/portfolio-planner/src/engine.ts

```diff
@@ -17,12 +17,31 @@ import type { SpectrumClient } from './spectrum-client.ts';
 import type { CosmosRestClient } from './cosmos-rest-client.ts';
 import { handleDeposit } from './plan-deposit.ts';
 import { submitAction } from './swingset-tx.ts';
+import { watchCCTPMint, type EVMChain } from './watch-cctp.ts';
 
 type CosmosEvent = {
   type: string;
   attributes?: Array<{ key: string; value: string }>;
 };
 
+type CCTPTransfer = {
+  amount: number;
+  caip: string;
+  receiver: string;
+};
+
+// Extended portfolio status type that includes CCTP transfers
+type PortfolioStatusWithCCTP = {
+  positionKeys: string[];
+  flowCount: number;
+  accountIdByChain: Record<string, `${string}:${string}:${string}`>;
+  depositAddress?: `${string}1${string}`;
+  targetAllocation?: Partial<Record<string, bigint>>;
+  pendingCCTPTransfers?: {
```

Comment:
As discussed in https://github.com/Agoric/agoric-private/issues/359#issuecomment-3157466606 , I don't expect this to be part of portfolio status. I expect it to go somewhere like...

`published.orchestration.subscriptions.subscription1234.`